gruntfile.js

```diff
@@ -10,7 +10,14 @@ module.exports = function(grunt)
                     basePath: 'themes',
                     declaration: false
                 },
-                src: ['src/default/assets/js/src/**/*.ts'],
+                src: [
+                    'src/default/assets/js/src/lib/**/*.ts',
+                    'src/default/assets/js/src/typedoc/Application.ts',
+                    'src/default/assets/js/src/typedoc/components/**/*.ts',
+                    'src/default/assets/js/src/typedoc/services/**/*.ts',
+                    'src/default/assets/js/src/typedoc/utils/**/*.ts',
```

Comment:
Cool, thanks. Sounds like a mess. Is there enough references in there to make this all work? I thought that controlled the output order, but I might have been wrong. What about if we add that you need at the beginning plus the glob? I'm sure it would be de-duped, but it's been a long time since I used Grunt often.